includes/admin/partials/form-options-debug.php

```diff
@@ -0,0 +1,200 @@
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+
+<div class="wrap">
+	<h1><?php esc_html_e( 'Algolia Debugging Tools', 'wp-search-with-algolia' ); ?></h1>
+	<p><?php esc_html_e( 'Use the tools below to inspect the plugin environment, check indexing status, and view recent logs.', 'wp-search-with-algolia' ); ?></p>
+
+	<h2><?php esc_html_e( 'Plugin Environment', 'wp-search-with-algolia' ); ?></h2>
+	<table class="widefat striped">
+		<tbody>
+			<tr>
+				<th><?php esc_html_e( 'Algolia API Key Present', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( get_option( 'algolia_application_id' ) && get_option( 'algolia_search_api_key' ) ? 'Yes' : 'No' ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'Application ID', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( get_option( 'algolia_application_id' ) ? get_option( 'algolia_application_id' ) : 'N/A' ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'Search API Key', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( get_option( 'algolia_search_api_key' ) ? get_option( 'algolia_search_api_key' ) : 'N/A' ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'Debug Mode', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( defined( 'WP_DEBUG' ) && WP_DEBUG ? 'Enabled' : 'Disabled' ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'Environment', 'wp-search-with-algolia' ); ?></th>
+				<td>
+				<?php
+				if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) ) {
+					echo esc_html( 'VIP (' . VIP_GO_APP_ENVIRONMENT . ')' );
+				} elseif ( defined( 'PANTHEON_ENVIRONMENT' ) ) {
+					echo esc_html( 'Pantheon (' . PANTHEON_ENVIRONMENT . ')' );
+				} elseif ( defined( 'WPE_ENV' ) ) {
+					echo esc_html( 'WPEngine (' . WPE_ENV . ')' );
+				} else {
+					echo esc_html( 'Local' );
+				}
+				?>
+				</td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'Plugin Version', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( defined( 'ALGOLIA_VERSION' ) ? ALGOLIA_VERSION : 'N/A' ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'PHP Version', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( PHP_VERSION ); ?></td>
+			</tr>
+			<tr>
+				<th><?php esc_html_e( 'WP Version', 'wp-search-with-algolia' ); ?></th>
+				<td><?php echo esc_html( get_bloginfo( 'version' ) ); ?></td>
+			</tr>
+		</tbody>
+	</table>
+
+	<h2><?php esc_html_e( 'Algolia Index Status', 'wp-search-with-algolia' ); ?></h2>
+	<?php
+	if ( isset( $this ) && property_exists( $this, 'plugin' ) ) {
+		$wpswa_plugin = $this->plugin;
+	} elseif ( function_exists( 'Algolia_Plugin_Factory' ) ) {
+		$wpswa_plugin = Algolia_Plugin_Factory::create();
+	} elseif ( class_exists( 'Algolia_Plugin_Factory' ) ) {
+		$wpswa_plugin = call_user_func( [ 'Algolia_Plugin_Factory', 'create' ] );
+	} else {
+		$wpswa_plugin = null;
+	}
+	if ( $wpswa_plugin && method_exists( $wpswa_plugin, 'get_indices' ) ) {
+		$wpswa_indices = $wpswa_plugin->get_indices();
```

Comment:
To reduce noise, let's change this `get_indices()` to this:

```
$wpswa_indices = $wpswa_plugin->get_indices(
	array(
		'enabled' => true,
	)
);
```
This way we're only looking at the ones presently in use.

Reply:
this would also allow for removing the "Enabled" column.